### ISSUE TITLE
models/nightly/thresholdModel1Factor5VariateWLS.R

### ISSUE THREAD

**Issue by jpritikin** [OpenMx/OpenMx#225]
Error SLSQP models/nightly/thresholdModel1Factor5VariateWLS.R ***
[1] "`a` not equivalent to `b`.\n1/1 mismatches\n[1] 1.63 - 0 == 1.63"

**Comment by tbates** [comment]
Running fine here under
> OpenMx version: 2.13.2.111 [GIT v2.13.2-111-gcf4f325]
R version: R version 3.6.1 (2019-07-05)
Platform: x86_64-apple-darwin15.6.0 
MacOS: 10.14.6

**Comment by tbates** [comment]
can this be closed?

**Comment by jpritikin** [comment]
No, these kind of issues exist to track occasional regressions over time.
